R/summaries.R

```diff
@@ -258,11 +265,23 @@ setMethod("countDominantTaxa", signature = c(x = "SummarizedExperiment"),
                      call. = FALSE)
             }
         }
+        # name check
+        if(!is.null(name)) {
+          if(is.character(name)==F) {
+            stop("'name' must be a character value", call. = F)
+          }
+          # Replace spaces with underscores to enhance usability
+          name <- gsub(" ", "_", name)
```

Comment:
Remove this, it's better to be exactly what user wants

Reply:
Needed to make some other changes instead, because column names with spaces caused problems